app/models/legal_aid_application.rb

```diff
@@ -217,19 +217,11 @@ def outgoing_types?
   end
 
   def generate_secure_id
-    transaction do
-      update!(
-        citizen_url_id: SecureRandom.uuid,
-        citizen_url_expires_on: CITIZEN_URL_EXPIRES_AFTER_IN_DAYS.days.from_now,
-      )
-
-      SecureData.create_and_store!(
-        legal_aid_application: { id: },
-        expired_at: (Time.current + SECURE_ID_DAYS_TO_EXPIRE.days).end_of_day,
-        # So each secure data payload is unique
-        token: SecureRandom.hex,
-      )
-    end
+    update!(
+      citizen_url_id: SecureRandom.uuid,
+      citizen_url_expires_on: CITIZEN_URL_EXPIRES_AFTER_IN_DAYS.days.from_now,
+    )
+    citizen_url_id
```

Comment:
I thought this would be a nice way to migrate to using the new approach, but it turns out we are currently calling `generate_secure_id` multiple times when we send an email to the applicant.

Before this worked because each URL would correspond to a new `SecureData` record that referenced the application.

However with the new approach, this means we send emails with URLs that contain `citizen_url_id` values that have since been overwritten).